### ISSUE TITLE
Project labels are shown to be deleted for every plan

### ISSUE THREAD

**Issue by stefanandres** [syseleven/terraform-provider-metakube#48]
In https://github.com/syseleven/terraform-provider-metakube/commit/7ea3bcf76a1fd62af2077e9f1826b55961e502ed was implemented that project labels do not get deleted for cluster.
For now only the deletion call is not sent, but the plan will now always show this change:

```
  # module.xxx.metakube_cluster.cluster will be updated in-place
  ~ resource "metakube_cluster" "cluster" {
        id                 = "XXX"
      ~ labels             = {
          - "xxx" = "" -> null
        }
        name               = "xxx"
```

When using CI reports this will confuse users why there always is a change

**Comment by furkhat** [comment]
Hi, I included a fix for this in v1.0.2

**Comment by stefanandres** [comment]
Works, thanks :)
